src/database/sequlize-handler/migrations/05_jobs_type.js

```diff
@@ -0,0 +1,15 @@
+const Sequelize = require('sequelize');
+
+module.exports.up = async (query, DataTypes) => {
+    let jobsTable = await query.describeTable('jobs');
+
+    if (!jobsTable.enabled) {
+        await query.addColumn(
+            'jobs', 'type',
+            Sequelize.DataTypes.STRING);
+    }
+};
```

Comment:
we would like to set a value for all the jobs in the new column `type=load_testing`

Reply:
done